integration-tests/tests/workspace-dashboard.js

```diff
@@ -172,10 +172,6 @@ const setAzureAjaxMockValues = async (testPage, namespace, name, workspaceDescri
     const workspaceSasTokenUrl = new RegExp(`api/workspaces/v1/${workspaceId}/resources/controlled/azure/storageContainer/(.*)/getSasToken`)
 
     window.ajaxOverridesStore.set([
-      {
```

Comment:
This is unrelated, but @aweng98 noticed it when working on switching tests to running against staging. This override was actually never used, as the namespace is incorrect (correct override is below at line 185).